inhouse_bot/game_queue/queue_handler.py

```diff
@@ -81,7 +81,7 @@ def add_player(
         session.merge(queue_player)
 
 
-def remove_player(player_id: int, channel_id: int = None):
+def remove_player(player_id: int, channel_id: int = None, role: str = None):
     """
     Removes the player from the queue in all roles in the channel
```

Comment:
We can change this comment since this functions sole purpose isnt this anymore.

Reply:
good catch